plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -650,4 +651,9 @@ private String getLongInClause(int start, int length)
                 .collect(joining(", "));
         return "orderkey IN (" + longValues + ")";
     }
+
+    private SqlExecutor onSqlServer()
```

Comment:
I am not a fan of this name (borrowed from PTs).
We already use it in oracle tests (`onOracle`). 
thoughts?

cc @kokosing

Reply:
`inRemoteDatabase`?

Reply:
`inRemoteDatabase` (`onRemoteDatabase`?) is good for generic tests where don't know where it is queried. I think this name is fine.

Can `TestingSqlServer` implement `SqlExecutor`?

Reply:
I thinking that a generic name would help working with tests, where there is a lot of copy pasting.

need to replace  "postgreSqlServer" with "sqlServer" or "mySqlServer" doesn't add value

Reply:
Ok, makes sense. 

In PT we are mixing many remotes at once so generic name does not apply there.

Reply:
Good point.

Changed to `onRemoteDatabase`